README.md

```diff
@@ -8,32 +8,32 @@
 ![Documentation Status](https://readthedocs.org/projects/feature-engine/badge/?version=latest)
 
 
-Feature-engine is a Python library with multiple transformers to engineer features for use in machine learning models. Feature-engine's transformers follow Scikit-learn functionality with fit() and transform() methods to first learn the transforming paramenters from data and then transform the data.
+Feature-Engine is a Python library with multiple transformers to engineer features for use in machine learning models. Feature-engine's transformers follow scikit-learn's functionality with fit() and transform() methods to first learn the transforming parameters from data and then transform the data.
 
 
-## Feature-engine features in the following resources:
```

Comment:
Feature-engine is an arbitrary name so for some reason I chose it to be F with capital and e with small one, and I wrote it that way in all docs. So I would prefer if we kept it this way. Could you reverse the change?

Reply:
Sure, I will change it back!